WordPress/src/main/res/layout/site_creation_theme_item.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.v7.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:card_view="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/theme_grid_card"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginLeft="@dimen/margin_extra_large"
+    android:layout_marginStart="@dimen/margin_extra_large"
+    android:layout_marginRight="@dimen/margin_extra_large"
+    android:layout_marginEnd="@dimen/margin_extra_large"
+    android:layout_marginBottom="@dimen/margin_extra_large"
+    android:foreground="?android:attr/selectableItemBackground"
+    android:clickable="true"
```

Comment:
If we add `android:focusable="true"` to the `CardView`, the view can be accessed with a keyboard also.

Reply:
Good call. Done with 93b87c1.

While at it, enabled the same behavior for the Category selection screen with 544b036. I think it's small enough change to be included in this PR.